src/openforms/formio/typing.py

```diff
@@ -40,3 +42,4 @@ class Component(TypedDict):
     label: str
     multiple: bool
     hidden: bool
+    defaultValue: JSONValue
```

Comment:
Should we maybe also add the `prefill` field, since it is used  in `_set_default_values` later?

Reply:
I'm adding them, but the type checks are bit inconsistent - as these keys may be absent. You can signal that with `typing.NotRequired`, but that's only included in Python 3.11+